### ISSUE TITLE
Linux Support

### ISSUE THREAD

**Issue by dotgodly** [poulfoged/redis-inside#10]

**Comment by Crozin** [comment]
Any chance for Linux support? What's stopping us from doing it?

**Comment by vmaleze** [comment]
Had the same need. 
Feel free to check => Linux compatibility / Remove old .Net Frameworks #12

**Comment by Crozin** [comment]
A work-around is proposed here #11

**Comment by vmaleze** [comment]
> A work-around is proposed here #11

Yeah I saw your PR as well. But in your implementation, you assume that redis-server is installed on the machine. 
The main purpose of this library is for tests. And you should be able to run those tests on any env and not depend on a redis-server that needs to be installed. You need to include the bin in the lib.  Otherwise, you are not running "inside"

**Comment by Crozin** [comment]
Yeah, this is why I've called it `UseExternalClient()` ;-) It's a little more difficult to provide a single "generic" linux binary for Redis. That's why I thought the use of external client might be still useful as this lib provides a newly created, isolated instance. Yes, the host must have Redis preinstalled to use this feature.

**Comment by Crozin** [comment]
Any update? :)

**Comment by laurentlbm-rc** [comment]
@poulfoged Any chance this could be added in?
